ci/run.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 set -ex
 
-export PATH=$PATH:$TRAVIS_BUILD_DIR/gradle/bin/
```

Comment:
it's still there, I just moved it past the declaration of the BUILD_DIR variable (line 10)